storybook/amp/config.js

```diff
@@ -1,26 +1,26 @@
 import React from 'react';
 import { ThemeProvider } from 'styled-components';
 import { addDecorator, configure } from '@storybook/react';
-import { setOptions } from '@storybook/addon-options';
+import { withOptions } from '@storybook/addon-options';
```

Comment:
The `setOptions` was deprecated by storybook so just switched it with `withOptions`. Ideally with v5 this will also go away but didn't wanted to screw up this PR. Will raise a separate PR for that